ocaml/testsuite/tests/typing-layouts/void_alpha.ml

```diff
@@ -2,8 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are unreviewed *)
```

Comment:
78 - refer to `void_rec` instead of `t_void` perhaps
333- potential place for layout history

Reply:
filed ticket